pkg/reconciler/v1alpha1/autoscaling/hpa/hpa.go

```diff
@@ -119,11 +120,12 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 		// This is important because the copy we loaded from the informer's
 		// cache may be stale and we don't want to overwrite a prior update
 		// to status with this stale state.
-	} else {
-		if _, err := c.updateStatus(pa); err != nil {
-			logger.Warnw("Failed to update pa status", zap.Error(err))
-			return err
-		}
+	} else if _, err := c.updateStatus(pa); err != nil {
+		logger.Warnw("Failed to update pa status", zap.Error(err))
+		return err
```

Comment:
do we want to skip this error?

Reply:
Hmm, most of the other resources surface a separate event for status updates, but this is missing that.  I'll add it.